### PR TITLE
chore(i18n): update es_ES translations

### PR DESCRIPTION
**Pull request by gboquizosanchez**
Following PR #2831, the es_ES translations have been updated.

@WedgeSparda, feel free to take a look if you're interested. 😉

### CODE REVIEW

File: lang/es_ES.json [hunk 1]
```diff
@@ -316,7 +316,7 @@
     "Title": "Título",
     "To upload an avatar, earn 250 points or wait until your account is at least 14 days old.": "Para subir un avatar, gana 250 puntos o espera hasta que tu cuenta tenga al menos 14 días.",
     "Toggle columns": "Alternar columnas",
-    "Type": "Type",
+    "Type": "Tipo",
```
[the remark this answers is not in the document]
Comment (gboquizosanchez) [reply lang/es_ES.json:319]: image

Confirmed!

And I have made some adjusts.

[the remark this answers is not in the document]
Comment (wescopeland) [reply lang/es_ES.json:319]: 👍 May be worth checking mobile on /demo/home and /games too. Try setting the device to iPhone SE on /games and different components will appear.

[the remark this answers is not in the document]
Comment (gboquizosanchez) [reply lang/es_ES.json:319]: image

This?

[the remark this answers is not in the document]
Comment (wescopeland) [reply lang/es_ES.json:319]: This stuff

Screenshot 2024-11-11 at 7 02 12 AM
Screenshot 2024-11-11 at 7 02 08 AM

**Review by wescopeland — APPROVED** [review]
Thanks for updating these! I'm happy to take what you already have. I'll leave this open for ~12 more hours in case there's any more changes you'd like to make or additional community feedback to respond to.

File: lang/es_ES.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (gboquizosanchez) [reply lang/es_ES.json:319]: image

In Spanish there is an issue here. After the comma, should be lower cased. :\ 

image

And in those as well.

But probably this should be solved in following issues ;)

**Comment by gboquizosanchez** [comment]
> Thanks for updating these! I'm happy to take what you already have. I'll leave this open for ~12 more hours in case there's any more changes you'd like to make or additional community feedback to respond to.

Perfect! Thanks!

**Pull request merged**